Gemfile

```diff
@@ -21,6 +21,7 @@ end
 manageiq_plugin "manageiq-automation_engine"
 manageiq_plugin "manageiq-content"
 manageiq_plugin "manageiq-providers-amazon"
+manageiq_plugin "amazon_ssa_support"
```

Comment:
Can you include a comment that this dependency is here temporarily and should be moved to `manageiq-providers-amazon` once it is officially released?